client/lib/abtest/index.js

```diff
@@ -96,7 +107,17 @@ ABTest.prototype.init = function( name ) {
 	this.defaultVariation = testConfig.defaultVariation;
 	this.variationNames = variationNames;
 	this.experimentId = name + '_' + variationDatestamp;
-	this.allowAnyLocale = testConfig.allowAnyLocale === true;
+
+	this.localeTargets = [ 'en' ];
+
+	if ( 'undefined' !== typeof testConfig.localeTargets ) {
+		if ( testConfig.localeTargets === false ) {
+			this.localeTargets = false;
+		} else {
+			this.localeTargets = testConfig.localeTargets;
+		}
+	}
```

Comment:
FWIW, IMO, `localeTargets: false` is not simple from a devex POV. It's not the end of the world. I do think it adds a bit of cognitive overload that seems avoidable.